app/globals.css

```diff
@@ -2,12 +2,6 @@
 @tailwind components;
 @tailwind utilities;
  
-html,
```

Comment:
Please do not remove lines of code that are written by other developers, if it is not included in the task requirements. This part of code is important and should stay as it is!

Reply:
Please make a new commit with the previous code from app/globals.css!

Reply:
I don't know how the previous code got deleted, but anyway I've created a PR with the the previous code for **app/global.css** from `bugfix/font_family_change`

Reply:
Okay I will check it!